airbyte-integrations/connectors/source-okta/source_okta/source.py

```diff
@@ -82,17 +83,22 @@ def cursor_field(self) -> str:
         pass
 
     def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
-        lowest_date = str(pendulum.datetime.min)
+        lowest_date = str(datetime.min)
```

Comment:
why this change from pendulum to datetime?

Reply:
pendulum.datetime.min wasn't resolving on my machine.

Reply:
you need to run tests with pyenv activated

Reply:
I reverted it back to pendulum